units/periodic_builds_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/require"
```

Comment:
I wonder why our lint or formatter allows this (and this comment isn't to fix it, there's probably all around our codebase), but the `testify/assert` package is imported down on L17 and then this `testify/require` is now it's own block on L8...